db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Methods.class/__methods__/postprovision.rb

```diff
@@ -10,9 +10,13 @@ def dump_stack_outputs(stack)
 
 $evm.log("info", "Starting Orchestration Post-Provisioning")
 
-service = $evm.root["service_template_provision_task"].destination
+task = $evm.root["service_template_provision_task"]
+service = task.destination
 stack = service.orchestration_stack
 
+# set onwer for the resources generated by the stack
+service.set_resources_owner
```

Comment:
@bzwei I think It would be better if this was a generic method name, like `post_provision_configure`.  The model method can then call whatever actions are required.  If needed the service model sub-classes could override the method and perform different actions.